contrib/bentoml/sources/docker-private-registry-proxy.yaml

```diff
@@ -0,0 +1,34 @@
+apiVersion: apps/v1
```

Comment:
@yetone could you please add an explanation why we need to DaemonSet? Question from Julius.